Doc/library/sqlite3.rst

```diff
@@ -273,14 +273,28 @@ Module functions and constants
    for the connection, you can set the *cached_statements* parameter. The currently
    implemented default is to cache 128 statements.
 
-   If *uri* is true, *database* is interpreted as a URI. This allows you
-   to specify options. For example, to open a database in read-only mode
-   you can use::
-
-       db = sqlite3.connect('file:path/to/database?mode=ro', uri=True)
-
-   More information about this feature, including a list of recognized options, can
-   be found in the `SQLite URI documentation <https://www.sqlite.org/uri.html>`_.
+   If *uri* is :const:`True`, *database* is interpreted as a
+   :abbr:`URI (Uniform Resource Identifier)` consisting of a scheme, an
+   authority, a path, and a query string. The scheme part *must* be ``"file:"``.
```

Comment:
The enumeration of URI components is either redundant with the understood form of file URIs, or potentially confusing ("Is this different than my understanding of URIs? What's an authority?").  Better would be to explain how to express a file path as a URI:
```suggestion
   :abbr:`URI (Uniform Resource Identifier)` with a file path and an optional query string.
   The scheme part *must* be ``"file:"``.  The path can be a relative or absolute file path.
```